.eslintrc.js

```diff
@@ -31,4 +31,24 @@ module.exports = {
       },
     ],
   },
+  overrides: [
+    {
+      files: ['**/*.controller.js'],
+      rules: {
+        '@typescript-eslint/camelcase': [
+          'warn',
+          {
+            allow: [
+              'fk_.*',
+              '.*_at',
+              'lat_.*',
+              'long_.*',
+              'answer_.*',
+              'max_players',
+            ],
```

Comment:
Great job. Please consider the last update in the [db ](https://github.com/HackYourFuture-CPH/fp-class12/blob/develop/dbdiagram_v4.png) that contains  `image_url `in the new Avatars table.

Reply:
Thanks Lucha, I think , of course I look through some documentations of `[https://eslint.org/docs/rules/camelcase](url)` and different issues on Github and StackOverflew, but since our problem here is specifically not allow it is natural to use verb allow not exceptions. 
About diagram I looked through last version  available on our repository at the time that I made the PR but as soon as I have new version I can adjust it :)

Reply:
Awesome. The new diagram is available here: https://github.com/HackYourFuture-CPH/fp-class12/blob/develop/dbdiagram_v4.png :)

Reply:
If you remove line 40-48 it will disable the camelcase rule entirely for files matching the file pattern `*.controller.js`, right? I believe that is what we want. We don't want to maintain a list of database table names in the eslint config. That's too much of a hazzle.

Reply:
I just tested it out. You need to do this:

    overrides: [
      {
        files: ['**/*.controller.js'],
        rules: {
          '@typescript-eslint/camelcase': ['off'],
          camelcase: ['off'],
        },
      },
    ],

The reason I have both `@typescript-eslint/camelcase` and `camelcase` rules added, is that eslint is good at figuring out if we are writing typescript or javascript. It's actually the `camelcase` rule that is relevant to us, but add both.

As you can see I've set them both to `off`, but only to file names that matches the file pattern.

Further, you can move this block to the `.eslintrc.js` within the server folder. The eslint config files override each other and we don't need the exception we just created on the client :)

Reply:
Done